src/main/java/net/sf/jabref/logic/msbib/MSBibMapping.java

```diff
@@ -52,34 +58,36 @@
         // MSBib only fields
         bibtexToMSBib.put(MSBIB_PREFIX + "numberofvolume", "NumberVolumes");
         bibtexToMSBib.put(MSBIB_PREFIX + "periodical", "PeriodicalTitle");
-        //bibtexToMSBib.put(MSBIB_PREFIX + "day", "Day");
-        //bibtexToMSBib.put(PREFIX + "accessed", "Accessed");
+        bibtexToMSBib.put(MSBIB_PREFIX + "day", "Day");
+        bibtexToMSBib.put(MSBIB_PREFIX + "accessed", "Accessed");
         bibtexToMSBib.put(MSBIB_PREFIX + "medium", "Medium");
         bibtexToMSBib.put(MSBIB_PREFIX + "recordingnumber", "RecordingNumber");
         bibtexToMSBib.put(MSBIB_PREFIX + "theater", "Theater");
         bibtexToMSBib.put(MSBIB_PREFIX + "distributor", "Distributor");
         bibtexToMSBib.put(MSBIB_PREFIX + "broadcaster", "Broadcaster");
         bibtexToMSBib.put(MSBIB_PREFIX + "station", "Station");
         bibtexToMSBib.put(MSBIB_PREFIX + FieldName.TYPE, "Type");
-        bibtexToMSBib.put(MSBIB_PREFIX + "patentnumber", "PatentNumber");
         bibtexToMSBib.put(MSBIB_PREFIX + "court", "Court");
         bibtexToMSBib.put(MSBIB_PREFIX + "reporter", "Reporter");
         bibtexToMSBib.put(MSBIB_PREFIX + "casenumber", "CaseNumber");
         bibtexToMSBib.put(MSBIB_PREFIX + "abbreviatedcasenumber", "AbbreviatedCaseNumber");
         bibtexToMSBib.put(MSBIB_PREFIX + "productioncompany", "ProductionCompany");
     }
 
+
     public static String getBibTeXEntryType(String msbibType) {
         final String defaultType = BibtexEntryTypes.MISC.getName();
 
         Map<String, String> entryTypeMapping = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
 
-        entryTypeMapping.put("Book", BibtexEntryTypes.BOOK.getName());
-        entryTypeMapping.put("BookSection", BibtexEntryTypes.INBOOK.getName());
-        entryTypeMapping.put("JournalArticle", BibtexEntryTypes.ARTICLE.getName());
-        entryTypeMapping.put("ArticleInAPeriodical", BibtexEntryTypes.ARTICLE.getName());
-        entryTypeMapping.put("ConferenceProceedings", BibtexEntryTypes.CONFERENCE.getName());
-        entryTypeMapping.put("Report", BibtexEntryTypes.TECHREPORT.getName());
+        entryTypeMapping.put("Book", BibLatexEntryTypes.BOOK.getName());
```

Comment:
Why isn't there a distinction between bibtex and biblatex? - Currently, the importers all do bibtex and the conversion to biblatex should be done afterwards (currently not implemented). For me, it would be also OK to parameterize the importers to generate bibtex or biblatex.